easybuild/easyconfigs/l/libxc/libxc-3.0.0-intel-2017a_runtest.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxc'
+version = '3.0.0'
+versionsuffix = '_runtest'
```

Comment:
@migueldiascosta hmm, not sure if this is worth the trouble...

If the tests work consistently, we can enable them, but I don't see much value in a separate easyconfig that enables tests that only work in some systems?